LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) MSK-INSIGHT
+Copyright (c) SIGNAL-DB
```

Comment:
Should it be just `SignalLDB` instead?

Reply:
hmm it all looks funky lol, too many CAPS - `SIGNALDB`?